lib/client.dart

```diff
@@ -275,6 +279,17 @@ class RouteImpl extends Route {
   }
 
   @override
+  void addRedirect({String name, Pattern path}) {
```

Comment:
At first glance, this API was a bit confusing to me. The `name` parameter makes it seem like you're naming the redirect, like you would name a route via `addRoute()` - but it's really defining which route to redirect to. Not sure I have a better alternative though.. only thing I thought of was `to` instead of `name`. It was only a second or two of hesitation so not a huge deal